### DISCUSSION TITLE
Why is the Edit file... view on iPad so small?

### DISCUSSION THREAD

**Discussion by IdanAdar** [community/community#128955] [Mobile] [answered]
### Select Topic Area

Question
Product Feedback

### Body

I have an iPad Air 13".
I wanted to see how the Edit file page looks like in the GitHub iOS app -- surprisingly it's super small. Why isn't there a button, or something, to increase the edit window to take advantage of the iPad screen size?! This makes editing not great at all, and the solution seems rather simple...

**Comment by IdanAdar** [comment]
At minimum, a full screen option is really basic to have...

![image](https://github.com/community/community/assets/7255583/aaeb5846-9414-4fc3-85fb-df9fc8fc9002)

**Comment by IdanAdar** [comment]
@mattruggio @stevepopovich @kangarruu hoping to get some of your attention here. :)

**Comment by esahmed21** [comment, marked as answer]
Hi @IdanAdar 👋 

Thank you for reaching out to us! I will create internal ticket for our team with your feedback!

**Reply by IdanAdar** [reply]
Thank you! But this doesn't answer my feedback... will I get an answer from your internal ticket?

**Reply by mattruggio** [reply]
Hey @IdanAdar, sorry, I marked this as an answer since it helped to generate an internal ticket which will now be used to track its progress. We can make a note in the internal ticket to follow back up on this original post once complete. Unfortunately I cannot see any estimated priority/timeline on it at this time though.

**Reply by IdanAdar** [reply]
Sure thing. I just hope the team sees the value in this. It does really make sense to make use of the screen space when editing code...

**Reply by IdanAdar** [reply]
Sorry, is it possible to get any potential updates here?

**Reply by mattruggio** [reply]
Hey @IdanAdar, I just checked and it is still in our internal queue awaiting some internal UI design feedback. As an iPad user myself, this one makes a ton of sense; I am also hoping this gets some 👀 on sooner than later.

**Reply by IdanAdar** [reply]
@mattruggio The latest build fixes this. Thank you very much!

I will be able to confirm this only once I get my Logitech combo touch, but I have a feeling that there's going to be a problem with the virtual keyboard. I notice that it is being poped up whenever I click on a line in edit mode, which is fine when you do not have a physical keyboard attached to your iPad. But when you do have it, one will want to suppress that...
